src/openqaoa-pyquil/backends/qaoa_pyquil_qpu.py

```diff
@@ -115,6 +116,8 @@ def __init__(
             cvar_alpha,
         )
         QAOABaseBackendCloud.__init__(self, device)
+        
+        self.gate_applicator = PyquilGateApplicator()
```

Comment:
comment for myself -- I need to convince myself why the `gate_applicator` needs to be an attribute.

Reply:
This is necessary because the wrapper needs to have access to the gate_applicator for the specific backend so that it can generate an empty circuit for the backend.